tests/test_users.py

```diff
@@ -74,7 +73,7 @@ def test_form_creation(self):
         form_endpoint = resp['random_like_string']
         self.assertIn(resp['random_like_string'], resp['submission_url'])
         self.assertEqual(1, Form.query.count())
-        self.assertEqual(Form.query.first().id, bigstring2int(resp['random_like_string']))
+        self.assertEqual(Form.query.first().id, HASHIDS_CODEC.decode(resp['random_like_string'])[0])
```

Comment:
Maybe you should use `Form.get_form_by_random_like_string(resp['random_like_string']).id` here?